201-zerto-zca-vpn/azuredeploy.json

```diff
@@ -0,0 +1,398 @@
+{  
+   "$schema":"http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+   "contentVersion":"1.0.0.0",
+   "parameters":{  
+      "vpnType":{  
+         "type":"string",
+         "metadata":{  
+            "description":"Route based (Dynamic Gateway) or Policy based (Static Gateway)"
+         },
+         "defaultValue":"RouteBased",
+         "allowedValues":[  
+            "RouteBased",
+            "PolicyBased"
+         ]
+      },
+      "localGatewayName":{  
+         "type":"string",
+         "defaultValue":"localGateway",
+         "metadata":{  
+            "description":"Name for gateway resource representing your local/on-prem gateway"
+         }
+      },
+      "localGatewayIpAddress":{  
```

Comment:
what if I don't have one of these?  will the deployment fail?

Reply:
all have default values now